skule_vote/backend/serializers.py

```diff
@@ -1,7 +1,43 @@
-from backend.models import Candidate, Election, ElectionSession
+from backend.models import Ballot, Candidate, Election, ElectionSession
 from rest_framework import serializers
 
 
+class BallotSerializer(serializers.BaseSerializer):
```

Comment:
This is outdated code :( 
It is missing the entire ballot submission endpoint and infra which includes a different ballot serializer. Need to pull from develop and rename this to BallotResultSerializer or something